.rubocop_todo.yml

```diff
@@ -62,13 +62,6 @@ Performance/FlatMap:
   Exclude:
     - 'app/models/maestrano/connector/rails/concerns/complex_entity.rb'
 
-# Offense count: 2
-# Cop supports --auto-correct.
-# Configuration parameters: AutoCorrect.
-Performance/HashEachMethods:
```

Comment:
@adamaziz15 I am having issues with this cop:
```
app/resources/maestrano/api/base_resource.rb:19:21: C: Performance/HashEachMethods: Use each_key instead of keys.each
```